src/components/app-root/index.tsx

```diff
@@ -35,9 +35,9 @@ class AppRootBase extends React.PureComponent<AppRootProps, {}> {
         bottomPane = <LoadData/>;
       } else {
         bottomPane = (
-          <SplitPane split="vertical" defaultSize={200}>
+          <SplitPane split="vertical" defaultSize={200} minSize={100}>
```

Comment:
Maybe add `maxSize={200}` as well to prevent users from making the pane wider than the default.

Reply:
Not sure why wider that the default is better?